modules/util.py

```diff
@@ -175,13 +175,23 @@ def get_files_from_folder(folder_path, exensions=None, name_filter=None):
             relative_path = ""
         for filename in sorted(files, key=lambda s: s.casefold()):
             _, file_extension = os.path.splitext(filename)
-            if (exensions is None or file_extension.lower() in exensions) and (name_filter is None or name_filter in _):
+            if (extensions is None or file_extension.lower() in extensions) and (name_filter is None or name_filter in _):
                 path = os.path.join(relative_path, filename)
                 filenames.append(path)
 
     return filenames
 
 
+def get_first_occurrence_in_folder(folder_path, filename):
```

Comment:
please check out the already existing model / lora file functions, which might be reusable here